tests/test_models.py

```diff
@@ -29,3 +29,16 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+def test_daily_stddev_integers():
+    """Test that mean function works for an array of positive integers."""
+    from inflammation.models import daily_mean
```

Comment:
Have called daily_stddev but imported daily_mean by mistake
```suggestion
    from inflammation.models import daily_stddev
```